operator-pipeline-images/operatorcert/entrypoints/detect_changed_operators.py

```diff
@@ -497,7 +572,10 @@ def detect_changes(
         **catalog_operators,
         "extra_files": list(non_operator_files),
     }
-    return result
+
+    _validate_result(result)
```

Comment:
It would be better to export these 2 functions from the detected changes. The validation and enrichment logic should be separated from the detect changes as it has very little in common.

Reply:
Function execution moved to the main() and tests separated.